Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2852,16 +2854,17 @@ extension BrowserViewController {
          Specific values here are from the WKContentView headers.
          https://github.com/JaviSoto/iOS9-Runtime-Headers/blob/master/Frameworks/WebKit.framework/WKContentView.h
         */
-        guard let webContentView = UIView.findSubViewWithFirstResponder(webView),
-            let input = webContentView.performSelector(Selector("inputAccessoryView")),
+        guard let webContentView = UIView.findSubViewWithFirstResponder(webView) else {
+            return
```

Comment:
Maybe throw in an `assertionFailure()` here before the `return`? That would cause us to crash on debug builds if `webContentView` is ever `nil`, which should never happen AFAICT.